README.md

```diff
@@ -1,14 +1,26 @@
-### Hexlet tests and linter status:
+# Brain Games
+
+## Package description:
+#### A small pack of fun games for your terminal
+
+## Package installation for Linux, macOS:
+### Run command 'python3 -m pip install --user dist/*.whl' or 'package-install' from package root catalog
```

Comment:
1. `make package-install`
2. if library is not built, there will be nothing to install (add build instructions)
3. add info about installation for development (poetry install/make install)